index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portafolio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+</head>
+<body>
+    <header class="header">
+        <span class="header__logo">My Logo</span>
+        <img src="./images/menu.svg" class="header__nav" alt="navigation icon">
+    </header>
```

Comment:
- [ ] Please make the logo a link that navigates back to the homepage. Users naturally expect the logo to take them back to the homepage when they click it 👍 
- [ ] Please enclose your hamburger button in a `<nav>` as it is part of your navigation

Reply:
I've Just done it. Thank you for your feedback, Codepantha!